packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain/useCustomFeeTokenBalances.ts

```diff
@@ -0,0 +1,27 @@
+import { useMemo } from 'react'
+
+import { Balances } from '../../../hooks/TransferPanel/useSelectedTokenBalances'
+import { useNativeCurrency } from '../../../hooks/useNativeCurrency'
+import { useNetworks } from '../../../hooks/useNetworks'
+import { useNetworksRelationship } from '../../../hooks/useNetworksRelationship'
+import { useBalances } from '../../../hooks/useBalances'
+
+export function useCustomFeeTokenBalances(): Balances {
```

Comment:
The name suggests this would only return custom fee token balances, but it also returns ETH if native currency is not custom.

Maybe call this method `useNativeTokenBalances`?